packages/palette/src/elements/Tags/Tags.tsx

```diff
@@ -24,28 +24,73 @@ export const Tags: React.FC<TagsProps> = ({
   displayNum = tags.length,
 }) => {
   const [expanded, setExpanded] = useState(false)
+  const [boxHeight, setBoxHeight] = useState("auto")
+  const flexContainer = useRef<HTMLDivElement>(null)
+  const animatingBox = useRef<HTMLDivElement>(null)
+  // after animating AnimatingBox height set back height to auto (to handle screen resize etc.)
+  useEffect(() => {
+    animatingBox.current.addEventListener("transitionend", e => {
+      const isAnimatingBoxTransition = Array.from(
+        (e.target as HTMLTextAreaElement).classList
+      ).some(elClass => elClass.indexOf("AnimatingBox") > -1)
+      if (isAnimatingBoxTransition) {
+        setBoxHeight("auto")
+      }
+    })
+  }, [])
   const sliceSize = expanded ? tags.length : displayNum
   const tagEls = tags.slice(0, sliceSize).map((tag, i) => {
     return <Tag key={i} {...tag} />
   })
+
+  /**
+   * By default AnimationBox height is auto. for open/close transition:
+   * 1. explicitly set the height to be the same as the flex container inside it,
+   * 2. display new tags (setExpanded(true))
+   * 3. set the height to the new flexContainer height so the animation triggers
+   */
+  const toggleMore = () => {
+    const BORDER_OFFSET = 2
```

Comment:
Without this 2 px `BORDER_OFFSET` the container height jumps a little bit when height is changed from px value to `auto` which I guessed it was because of top/bottom borders (?!).